packages/dev/core/src/Maths/math.plane.ts

```diff
@@ -137,6 +137,18 @@ export class Plane {
         return this;
     }
 
+    /**
+     * Updates the current Plane from an origin point and a normal.
+     * @param origin origin of the plane to be constructed
+     * @param normal the normalized normals of the plane to be constructed
+     * @returns the updated Plane.
+     */
+    public copyFromPositionAndNormal(origin: DeepImmutable<Vector3>, normal: DeepImmutable<Vector3>) {
+        this.normal = normal;

```

Comment:
as the method is called copy, should it not set the value only here to prevent unintended side effects ? like this.normal.copyFrom(normal)